src/main/java/com/gsheet/student/config/GoogleAuthConfig.java

```diff
@@ -2,27 +2,40 @@
 
 import com.gsheet.student.service.AuthTokenProvider;
 import com.gsheet.student.service.GoogleAuthTokenProvider;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.io.ClassPathResource;
 import org.springframework.core.io.Resource;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
 @Configuration
+@ConfigurationProperties(prefix = "google.auth")
 public class GoogleAuthConfig {
 
-    @Value("${google.auth.credential.file-path}")
     private String credentialFilePath;
 
+    private List<String> scopes = new ArrayList<>();
```

Comment:
If you need array type maybe it is better to initialise values with array and use it on Line27?